packages/medusa/src/utils/countries.ts

```diff
@@ -1,404 +1,2 @@
-    alpha3: "SJM",
-    numeric: "744",
-  },
-  { alpha2: "SZ", name: "Swaziland", alpha3: "SWZ", numeric: "748" },
-  { alpha2: "SE", name: "Sweden", alpha3: "SWE", numeric: "752" },
-  { alpha2: "CH", name: "Switzerland", alpha3: "CHE", numeric: "756" },
-  { alpha2: "SY", name: "Syrian Arab Republic", alpha3: "SYR", numeric: "760" },
-  {
-    alpha2: "TW",
-    name: "Taiwan, Province of China",
-    alpha3: "TWN",
-    numeric: "158",
-  },
-  { alpha2: "TJ", name: "Tajikistan", alpha3: "TJK", numeric: "762" },
-  {
-    alpha2: "TZ",
-    name: "Tanzania, United Republic of",
-    alpha3: "TZA",
-    numeric: "834",
-  },
-  { alpha2: "TH", name: "Thailand", alpha3: "THA", numeric: "764" },
-  { alpha2: "TL", name: "Timor Leste", alpha3: "TLS", numeric: "626" },
-  { alpha2: "TG", name: "Togo", alpha3: "TGO", numeric: "768" },
-  { alpha2: "TK", name: "Tokelau", alpha3: "TKL", numeric: "772" },
-  { alpha2: "TO", name: "Tonga", alpha3: "TON", numeric: "776" },
-  { alpha2: "TT", name: "Trinidad and Tobago", alpha3: "TTO", numeric: "780" },
-  { alpha2: "TN", name: "Tunisia", alpha3: "TUN", numeric: "788" },
-  { alpha2: "TR", name: "Turkey", alpha3: "TUR", numeric: "792" },
-  { alpha2: "TM", name: "Turkmenistan", alpha3: "TKM", numeric: "795" },
-  {
-    alpha2: "TC",
-    name: "Turks and Caicos Islands",
-    alpha3: "TCA",
-    numeric: "796",
-  },
-  { alpha2: "TV", name: "Tuvalu", alpha3: "TUV", numeric: "798" },
-  { alpha2: "UG", name: "Uganda", alpha3: "UGA", numeric: "800" },
-  { alpha2: "UA", name: "Ukraine", alpha3: "UKR", numeric: "804" },
-  { alpha2: "AE", name: "United Arab Emirates", alpha3: "ARE", numeric: "784" },
-  { alpha2: "GB", name: "United Kingdom", alpha3: "GBR", numeric: "826" },
-  { alpha2: "US", name: "United States", alpha3: "USA", numeric: "840" },
-  {
-    alpha2: "UM",
-    name: "United States Minor Outlying Islands",
-    alpha3: "UMI",
-    numeric: "581",
-  },
-  { alpha2: "UY", name: "Uruguay", alpha3: "URY", numeric: "858" },
-  { alpha2: "UZ", name: "Uzbekistan", alpha3: "UZB", numeric: "860" },
-  { alpha2: "VU", name: "Vanuatu", alpha3: "VUT", numeric: "548" },
-  { alpha2: "VE", name: "Venezuela", alpha3: "VEN", numeric: "862" },
-  { alpha2: "VN", name: "Viet Nam", alpha3: "VNM", numeric: "704" },
-  {
-    alpha2: "VG",
-    name: "Virgin Islands, British",
-    alpha3: "VGB",
-    numeric: "092",
-  },
-  { alpha2: "VI", name: "Virgin Islands, U.S.", alpha3: "VIR", numeric: "850" },
-  { alpha2: "WF", name: "Wallis and Futuna", alpha3: "WLF", numeric: "876" },
-  { alpha2: "EH", name: "Western Sahara", alpha3: "ESH", numeric: "732" },
-  { alpha2: "YE", name: "Yemen", alpha3: "YEM", numeric: "887" },
-  { alpha2: "ZM", name: "Zambia", alpha3: "ZMB", numeric: "894" },
-  { alpha2: "ZW", name: "Zimbabwe", alpha3: "ZWE", numeric: "716" },
-  { alpha2: "AX", name: "Åland Islands", alpha3: "ALA", numeric: "248" },
-]
+import { defaultCountries, Country } from "@medusajs/utils"
```

Comment:
It seems `defaultCountries` and `defaultCurrencies` were not even used from the utils package